library/java/net/openid/appauth/AuthorizationServiceConfiguration.java

```diff
@@ -224,6 +224,28 @@ public static void fetchFromIssuer(@NonNull Uri openIdConnectIssuerUri,
         fetchFromUrl(buildConfigurationUriFromIssuer(openIdConnectIssuerUri), callback);
     }
 
+    /**
+     * Fetch an AuthorizationServiceConfiguration from an OpenID Connect issuer URI, using
```

Comment:
[]()```
- # _**@**_``
```SGNrftalo

Reply:
#131

Reply:
Sorry I'm not sure if you are writing meaningful words